src/cart/crt_group.c

```diff
@@ -3046,50 +3042,44 @@ crt_group_secondary_rank_add_internal(struct crt_grp_priv *grp_priv,
 
 	/* Add entry to lookup table. Secondary group table contains ranks */
 	rm_s2p = crt_rank_mapping_init(sec_rank, prim_rank);
-	if (!rm_s2p) {
-		D_ERROR("Failed to allocate entry\n");
```

Comment:
Were these showing up as errors?

FYI - the test checks for new errors that don't end in the DF_RC/DP_RC print with -DER_NOMEM, so you don't need to remove all errors from the failure path, just include what the error is.

Reply:
Inside the crt_rank_mapping_init() only one error can happens. And this is in D_ALLOC_PTR() which will report about this. 

I removed only errors which duplicates errors from D_ALLOC*() macros.

Reply:
Great.  Why is this code-path being taken now as I don't think this error was showing before?